lib/validators.js

```diff
@@ -155,6 +155,17 @@ validator.integer = function(ctx, options, value) {
   }
 }
 
+validator.values = function(ctx, options, value) {
```

Comment:
I think the validator should be called `enum` and `items` instead of `validValues`, what do you think?